README.md

```diff
@@ -42,19 +42,7 @@ Follow these instructions to prepare your system for development.
 5. Run the client.
 
    ```shell
-   $ sudo PYTHONPATH=./src BYPASS_GPG=True EGG=../insights-core python src/insights_client/__init__.py --no-gpg --help
-   ```
-
-   *Note: `BYPASS_GPG` skips the verification on Client side, `--no-gpg` disables it on Core side.*
-
-6. To build an insights-core egg from source, run `build_client_egg.sh` from the insights-core repo.
-
-   ```shell
-   $ cd ../insights-core
-   $ bash build_client_egg.sh
-   $ # File `insights.zip` gets created in the current directory
-   $ cd ../insights-client
-   $ # To use the zip file as an egg, pass `EGG=../insights-core/insights.zip`
+   $ sudo PYTHONPATH=./src:../insights-core python src/insights_client/__init__.py --help
```

Comment:
Previous steps used `python3`, so maybe it would be better to be consistent with that